extra/alacritty.man

```diff
@@ -49,7 +49,15 @@ Defines the window class hint on Linux [default: Alacritty,Alacritty ]
 Command and args to execute (must be last argument)
 .TP
 \fB\-\-config\-file\fR <config\-file>
-Specify alternative configuration file [default: $XDG_CONFIG_HOME/alacritty/alacritty.yml]
+Specify alternative configuration file
+
+Alacritty looks for the configuration file at the following paths:
+    1. $XDG_CONFIG_HOME/alacritty/alacritty.yml
+    2. $XDG_CONFIG_HOME/alacritty.yml
+    3. $HOME/.config/alacritty/alacritty.yml
+    4. $HOME/.alacritty.yml
+
+On Windows, the configuration file is located at %APPDATA%\\alacritty\\alacritty.yml.
```

Comment:
Are man pages a thing on Windows?

Reply:
I mean you can read them if you have a program to look at them. So if you're running bash inside of Alacritty, there's a good chance that `man` is available too.

But since Alacritty still runs on Windows, even when the shell offers posix things, the config file will still be at the Windows location.

Reply:
Let's keep it then.